src/cmd-build

```diff
@@ -314,7 +341,16 @@ cat > tmp/meta.json <<EOF
 }
 EOF
 
-echo '{ "images": {} }' > tmp/images.json
+cat > tmp/images.json <<EOF
+{
+  "images": {
+    "ostree": {
```

Comment:
This is tough. I think it can make sense, but at the same time, I see the `ostree-commit-object` as more internal than the rest of the artifacts in that list. E.g. we really don't expect folks to download that file. Also, the checksum is already covered by `ostree-commit`, so the only win we'd derive is encoding its name in case we want to change it (but again, people shouldn't really need to download it).

Maybe another way to say this is that I see the OSTree tarfile as more "stable" whereas the commit object is more of an implementation detail. (And of course, the tarfile includes the commit object too.)

Reply:
I'm cool with that. I trust your opinion on this